client/executor/wasmtime/Cargo.toml

```diff
@@ -21,11 +21,11 @@ sp-wasm-interface = { version = "2.0.0-rc4", path = "../../../primitives/wasm-in
 sp-runtime-interface = { version = "2.0.0-rc4", path = "../../../primitives/runtime-interface" }
 sp-core = { version = "2.0.0-rc4", path = "../../../primitives/core" }
 sp-allocator = { version = "2.0.0-rc4", path = "../../../primitives/allocator" }
-wasmtime = { package = "substrate-wasmtime", version = "0.16.0-threadsafe.4" }
-wasmtime-runtime = { package = "substrate-wasmtime-runtime", version = "0.16.0-threadsafe.4" }
-wasmtime-environ = "0.16"
-cranelift-wasm = "0.63"
-cranelift-codegen = "0.63"
+wasmtime = { package = "substrate-wasmtime", git = "https://github.com/paritytech/wasmtime", branch = "update-upstream" }
+wasmtime-runtime = { git = "https://github.com/paritytech/wasmtime", branch = "update-upstream" }
+wasmtime-environ = { git = "https://github.com/paritytech/wasmtime", branch = "update-upstream" }
+cranelift-wasm = { git = "https://github.com/paritytech/wasmtime", branch = "update-upstream" }
+cranelift-codegen = { git = "https://github.com/paritytech/wasmtime", branch = "update-upstream" }
```

Comment:
Need to put this onto a released version once tested or we can't publish this crate...

Reply:
I will change this once you publish `substrate-wasmtime` from `paritytech/wasmtime/update-upstream`

Reply:
is this published now? Did we merge this with github links?

Reply:
Not published, can you publish substrate-wasmtime 0.19 like I asked above?

I will add prepare update pr after

Reply:
ah, sorry, didn't see you wanted me to do this. on it.

Reply:
@NikVolf substrate-wasmtime 0.19.0 is published.